APIs/controllers/authCtrl.js

```diff
@@ -225,8 +225,175 @@ async function register(req, res) {
     }
 
 }
+function reset(req, res){
+    if (!req.body.password) {
+        res.status(500).json({
+            success: false,
+            message: "password is required"
+        });
+        return;
+    }
+    var reset_token = req.body.reset_token;
```

Comment:
@AyonPal please add a check for `req.body.reset_token` as it's needed to make a DB query